components/eamxx/src/dynamics/homme/eamxx_homme_iop.cpp

```diff
@@ -144,7 +156,12 @@ advance_iop_subsidence(const KT::MemberType& team,
     Pack delta_tracer_k, delta_tracer_km1;
     for (int iq=0; iq<n_q_tracers; ++iq) {
       auto s_delta_tracer = Kokkos::subview(s_delta_Q, iq, Kokkos::ALL());
-      ekat::index_and_shift<-1>(s_delta_tracer, range_pack_m1, delta_tracer_k, delta_tracer_km1);
+      ekat::index_and_shift<-1>(s_delta_tracer, range_pack_for_m1_shift, delta_tracer_k, delta_tracer_km1);
+      if (at_top.any() or at_bot.any()) {
+        delta_tracer_k.set(at_top, s_delta_tracer(0));
```

Comment:
This `at_top` seems like a typo. I guess it should be `at_bot`?

Reply:
I think this is correct? At top (at least how I'm using it) is level=0, and at bottom is level=nlevs-1. The top and bottom being reversed index always confuses me though, so I could be missing something..

Reply:
Ah right. sorry.